app/main.py

```
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file, destination_file = parts[1], parts[2]
+    if parts[1] == parts[2]:
+        return
+
+    if not os.path.exists(source_file):
+        return
+
+    try:
+        with (open(source_file, "r") as file1,
+              open(destination_file, "w") as file2
+              ):
+            file2.write(file1.read())
+    except Exception:
+        pass
```

Catching a general `Exception` is not recommended as it can suppress unexpected errors. Consider catching more specific exceptions, such as `IOError` or `OSError`, to handle file-related errors appropriately.